src/write/elf.rs

```diff
@@ -597,6 +597,9 @@ impl Object {
                             (RelocationKind::Relative, RelocationEncoding::Generic, 16) => {
                                 elf::R_AARCH64_PREL16
                             }
+                            (RelocationKind::Relative, RelocationEncoding::AArch64Call, 26) => {
+                                elf::R_AARCH64_CALL26
+                            }
```

Comment:
This is good, but while you're here it wouldn't hurt to handle `PltRelative` too.